rbac/management/relation_replicator/outbox_replicator.py

```diff
@@ -123,10 +124,12 @@ def _build_replication_event(self, event: ReplicationEvent) -> ReplicationEventP
 
         add_json: list[dict[str, Any]] = []
         for relation in event.add:
+            RelationTuple.validate_message(relation)
```

Comment:
We don't need this check in the replicator, it is too late in the process. As long as you have it in the create relationship, should be fine

Reply:
right, we should always use create_relationship, and add the validation in that layer. another iteration checking the tuples here is not efficient